L1Trigger/L1CaloTrigger/python/L1TowerCalibrationProducer_cfi.py

```diff
@@ -28,7 +28,9 @@
     hfSF = cms.double(1.0),
     debug = cms.bool(False),
     skipCalibrations = cms.bool(False),
-    l1CaloTowers = cms.InputTag("L1EGammaClusterEmuProducer",),
+    #l1CaloTowers = cms.InputTag("L1EGammaClusterEmuProducer",),
+    #l1CaloTowers = cms.VInputTag(cms.InputTag("L1EGammaClusterEmuProducer","L1CaloTowerCollection","")),
```

Comment:
Do we really need these? I mean: it's useful because a switch between the two may be done in the future? If not I would remove them.